docs/guide/globalaccelerator/aga-controller.md

```diff
@@ -0,0 +1,289 @@
+# AWS Global Accelerator Controller
+
+## Introduction
+
+AWS Global Accelerator (AGA) is a networking service that improves the performance of users' traffic by up to 60% using Amazon Web Services' global network infrastructure. The AWS Load Balancer Controller (LBC) extends its capabilities with a new AGA Controller that allows users to declaratively manage AWS Global Accelerator resources using Kubernetes Custom Resource Definitions (CRDs).
+
+This feature bridges a significant operational gap - traditionally, AWS Global Accelerator must be manually created and managed through the AWS console or CLI, creating operational overhead and lacking integration with Kubernetes-native workflows. The AGA Controller eliminates this gap by enabling full lifecycle management of Global Accelerator resources directly from your Kubernetes cluster.
+
+## Architecture
+
+The AGA Controller operates as a continuous reconciliation loop within the AWS Load Balancer Controller, watching for changes to the `GlobalAccelerator` CRD. When it detects a change, it:
+
+1. Translates the desired state defined in the CRD into corresponding AWS Global Accelerator API calls
+2. Automatically discovers resources like Elastic Load Balancers (ELBs) referenced by Kubernetes Services, Ingresses, and Gateway APIs
+3. Maintains and updates the status of the AGA resources in the CRD
+4. Handles cleanup and resource deletion when objects are removed
+
+## Key Features
+
+### Monolithic CRD Design
+
+The controller uses a single `GlobalAccelerator` resource to manage the entire AGA hierarchy, including:
+
+- The accelerator itself
+- Listeners with protocol and port range configurations
+- Endpoint groups with region and traffic dial settings
+- Endpoints from different Kubernetes resource types
```

Comment:
I'd highlight AWS Load Balancer Controller Ingress, Service, and Gateway resources specifically here as supported endpoint types